### PR TITLE
Senri dental clinic section on About-us child page

### LINKED ISSUE #26: Add styling to Sunstar senri dental clinic sction sction
image

### PR DESCRIPTION
**Pull request by JiangLong2019**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #26 

Test URLs:
- Original: https://www.sunstar-foundation.org/en/about-us/preventive-care
- Before: https://main--sunstar-foundation--hlxsites.hlx.live/en/about-us/preventive-care
- After: https://senri-dental-clinic-sction--sunstar-foundation--hlxsites.hlx.live/en/about-us/preventive-care

- [ ] New Blocks introduced in this PR
      If yes, please provide details below

Block Name    | Documentation
------------- | -------------

- [x] New variations introduced in this PR

Variation Name    | Documentation
------------- | -------------
Columns(No-margin), Section(large-padded)  |

### CODE REVIEW

File: blocks/columns/columns.css [hunk 1]
```diff
@@ -6,6 +6,10 @@
   margin: 2rem 0 0;
 }
 
+.columns.no-margin {
```
[the remark this answers is not in the document]
Comment (JiangLong2019) [reply blocks/columns/columns.css:9]: @sdmcraft Great, we had this already! Let me remove the variant.

**Comment by JiangLong2019** [comment]
Hi @bosschaert, yeah, that's because I changed the H2 font size for columns block to make it as same as original site. I think it looks better, what do you think?

Before: --heading-font-size-mxl
After: --heading-font-size-ml
>Some visual changes were detected. @JiangLong2019 did you check if these are acceptable?

**Comment by ehrro** [comment]
Hello @JiangLong2019 it looks better. 


Would it be possible to align the About us text with the container?

image

Thank you.

**Comment by JiangLong2019** [comment]
>Would it be possible to align the About us text with the container?

@ehrro Sure, it will be fixed on issue #42. If there in no other concerns, can you please approve the current PR?

File: styles/styles.css [hunk 2]
```diff
@@ -749,6 +749,15 @@ main .section.narrower > .tab-item.active > .section-container {
   max-width: 45rem;
 }
 
+/* section - large-padded */
+main .section.large-padded {
```
Comment (sdmcraft) [inline styles/styles.css:753]: These paddings should be applied to `section-container` level. `section` by itself should occupy the full space.

File: styles/styles.css [hunk 3]
```diff
@@ -749,6 +749,15 @@ main .section.narrower > .tab-item.active > .section-container {
...
+  padding: 4rem 1rem;
+}
+
+main .section.large-padded h2 {
```
Comment (sdmcraft) [inline styles/styles.css:757]: How about we just keep the default `h2` style if the difference is acceptable? /cc @ehrro

File: styles/styles.css [hunk 4]
```diff
@@ -1029,6 +1038,11 @@ main .block.centered > div {
     font-size: var(--heading-font-size-mxl);
   }
 
+  /* section - large-padded */
+  main .section.large-padded {
+    padding: 5rem 0;
```
Comment (sdmcraft) [inline styles/styles.css:1043]: These paddings should be applied to section-container level. section by itself should occupy the full space.

File: styles/styles.css (hunk 3 above)
Comment (JiangLong2019) [reply styles/styles.css:757]: I thought the same, maybe we should merge the base typography of sunstar.com

File: styles/styles.css (hunk 2 above)
Comment (JiangLong2019) [reply styles/styles.css]: Fixed

File: styles/styles.css (hunk 4 above)
Comment (JiangLong2019) [reply styles/styles.css]: Fixed

File: styles/styles.css (hunk 3 above)
Comment (sdmcraft) [reply styles/styles.css:757]: I am not sure if this site requires a typography update or it look good in its current state. If an update is needed, we can cherry-pick [this commit](https://github.com/hlxsites/sunstar/commit/0d8af249631ab6678337a8c4b8e47f13b3d45e53)
@ehrro wdyt?

**Review by sdmcraft — APPROVED** [review]

**Pull request merged**